static/js/education-actions.js

```diff
@@ -3,6 +3,22 @@ $(function () {
   const $openAll = $('.govuk-accordion__open-all')
   const $printLink = $('.print-link')
 
+  const $neurodivergenceAccordion = $('#prisoner-accordion-heading-5')
+  const $linkToNeurodiversityTab = $('#view-neurodivergence-link')
+
+  $linkToNeurodiversityTab.on('click', () => {
```

Comment:
Not seen this before - so when you click on a link to a tab it trigger a click on the accordian?

Reply:
Confirmed